resource.cpp

```diff
@@ -1178,13 +1190,28 @@ bool ResourceItem::setValue(const QVariant &val, ValueSource source)
             return true;
         }
     }
-    else
+    else if (m_rid->type == DataTypeReal)
```

Comment:
Do we need to also support min/max ranges as for integer values?

Reply:
See above.  I don’t think we have any API writeable `DataTypeReal` items yet.